resources/aws_ec2.go

```diff
@@ -111,6 +113,36 @@ const (
 	awsEc2EventInstanceExists
 )
 
+// bootstrapScript is a bash script template for installing mgmt and connecting
+// back to the seed specified in the definition.
+const bootstrapScript string = `#!/bin/bash
+if [ -n "$(command -v apt)" ]
```

Comment:
what's command?
what about `which` and `test -z`, eg: https://github.com/purpleidea/mgmt/blob/master/misc/make-deps.sh#L14

(which one is preferable?)

Reply:
afaik, `command` is built into the shell whereas `which` is an external binary. Truthfully I don't think it really matters which one we use.

Reply:
Yeah, that's cool if this works! I had never even seen this before :) Is it something that works on bash and sh ?

Reply:
It works in sh for me.